lib/elixir/lib/stream.ex

```diff
@@ -1404,6 +1401,22 @@ defmodule Stream do
     do: %{lazy | funs: [fun | funs], accs: [acc | accs], done: done}
   defp lazy(enum, acc, fun, done),
     do: %Stream{enum: enum, funs: [fun], accs: [acc], done: done}
+
+  defp do_interval({:first_time, interval}) do
+    { 0, { 1, now_in_ms(), interval }}
+  end
+
+  defp do_interval({count, previous_time, interval}) do
+    then = previous_time + interval
+    now  = now_in_ms()
+
+    if then > now, do: Process.sleep(then - now)
+
+    {count, { count + 1, then, interval }}
+  end
+    
+  defp now_in_ms(),
```

Comment:
The function name is better as now_in_ms, because that leaves the implementation out of it. The fact that it's in ms is required, because we pass it to sleep(). The fact that its monotonic time is not relevant—the only requirement on the function is that now_in_ms values can be subtracted to give a difference in ms. How it is calculated may well change in the underlying Erlang time system changes (again).

I'll remind you that the original function's parameter was called `n` :)

And folks—if the styling is so important, then perhaps write a tool that puts stuff into it :)